wsd/HostUtil.cpp

```diff
@@ -148,6 +150,14 @@ std::string HostUtil::getNewUri(const Poco::URI& uri)
     {
         newUri.setAuthority(value);
     }
+    else
+    {
+        const std::string val = Util::getValue(hostList, newUri.getHost());
+        if (!val.empty() && val.find('*') != std::string::npos)
```

Comment:
I wonder if we can use something better to check it was a regex match - what about to compare 'val' to the newUri.getHost()? If they are not equal, it was a regex match, and the pair should be added to AliasHosts, I think - can you please check?